src/tools/cephfs_mirror/Mirror.cc

```diff
@@ -496,31 +494,33 @@ void Mirror::update_fs_mirrors() {
   dout(20) << dendl;
 
   auto now = ceph_clock_now();
+
   double blocklist_interval = g_ceph_context->_conf.get_val<std::chrono::seconds>
     ("cephfs_mirror_restart_mirror_on_blocklist_interval").count();
-  bool check_blocklist = blocklist_interval > 0 && ((now - m_last_blocklist_check) >= blocklist_interval);
-
   double failed_interval = g_ceph_context->_conf.get_val<std::chrono::seconds>
     ("cephfs_mirror_restart_mirror_on_failure_interval").count();
-  bool check_failure = failed_interval > 0 && ((now - m_last_failure_check) >= failed_interval);
```

Comment:
maybe `(failed_interval > 0 && (mirror_action.fs_mirror->get_failed_ts() - now) > failed_interval)` can be assigned to this bool variable; just for better readability